benchmark/createTestCases.js

```diff
@@ -50,20 +50,34 @@ function createTree(fs, count, folder) {
 	let remaining = count - 1;
 
 	function make(prefix, count, depth) {
-		if(count === 0) {
+		if (count === 0) {
 			fs.writeFileSync(`${folder}/${prefix}.js`, `export default 1;\n${avgJs}`);
 		} else {
 			const list = [];
-			for(let i = 0; i < count; i++) {
-				if(remaining-- <= 0) break;
-				if(depth <= 4 && i >= 3 && i <= 4) {
-					list.push(`const module${i} = import("./${prefix}-${i}");\ncounter += module${i};`);
+			for (let i = 0; i < count; i++) {
+				if (remaining-- <= 0) break;
+				if (depth <= 4 && i >= 3 && i <= 4) {
+					list.push(
+						`const module${i} = import("./${prefix}-${i}");\ncounter += module${i};`
+					);
 				} else {
-					list.push(`import module${i} from "./${prefix}-${i}";\ncounter += module${i};`);
+					list.push(
+						`import module${i} from "./${prefix}-${i}";\ncounter += module${i};`
+					);
 				}
-				make(`${prefix}-${i}`, depth > 4 || count > 30 ? 0 : count + depth + i ** 2, depth + 1);
+				make(
+					`${prefix}-${i}`,
+					// eslint-disable-next-line node/no-unsupported-features
```

Comment:
Why is it needed? Could you remove this and fix the error?

Reply:
So far I recall the `**` operator is not supported in our target version of Node. I'll have a look into it

Reply:
You are right. Could you switch to `Math.pow`?

Reply:
Yeah, I was thinking of the same, that should work 👍